src/javascript/detectors/os_command_injection/os_command_injection.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=os-command-injection@v1.0 defects=1}
+var express = require('express')
+var app = express()
+
+function osCommandInjectionNoncompliant() {
+    app.get('/user/:id', function (request, response) {
```

Comment:
Can we keep the syntax consistent between noncompliant and compliant examples?
- Below example uses arrow function notation
- Argument names are `req` and `res`